lib/cylc/network/client.py

```diff
@@ -190,8 +190,13 @@ def __init__(self, host, port, encode_method, decode_method, secret_method,
             raise ClientError(response['error'])
 
     def serial_request(self, command, args=None, timeout=None):
-        return asyncio.run(
-            self.async_request(command, args, timeout))
+        # TODO: replace by asyncio.run when minimum req is py37
```

Comment:
Added this todo because the `asyncio.run` actually does a bit more. It checks whether the loop created is the only one, as it is not expected to use one loop while there is already an asyncio loop in the same thread. It also collects events when debugging, and on shutdown of the loop tries to cancel any remaining futures enqueued. So not exactly the same as old code used to do, but same plus good improvements that would be nice for us to use too IMO.